inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -172,6 +175,21 @@ public ActiveLearningSidebar(String aId, IModel<AnnotatorState> aModel,
         super(aId, aModel, aActionHandler, aJCasProvider, aAnnotationPage);
 
         annotationPage = aAnnotationPage;
+        project = aModel.getObject().getProject();
+        user = aModel.getObject().getUser();
+
+        //set the user states to the ActiveLearningSidebar fields
+        sessionActive = activeLearningService.getSessionActive(user, project);
```

Comment:
The components should read/write directly to the state, not to local copies of the values. So all these variables should be removed.

Reply:
Every time the active learning sidebar is opened, all the methods here are loaded. And I think this is the only time that these fields get value from the saved state. That's why I let these components read here.